lib/config.py

```diff
@@ -64,6 +64,7 @@ def load_env(self):
         * `DISPLAY_PROGRESS` (boolean): whether or not to display training progress in logs (_e.g. false_)
         * `MAX_RECS` (int): How many recommendations to save to the database for each article (_e.g. 20_)
         * `DAYS_OF_DATA` (int): How many days worth of data to fetch and train on (_e.g. 28_)
+        * `SS_ENCODER` (str): name of pretrained model used to create article text embeddings, for semantic-similarity method (https://www.sbert.net/docs/pretrained_models.html)
```

Comment:
🌱 This could be an `enum` instead of a string, thereby making sure its always a valid model name. Not a change to make, just an idea for the future, and multiple ways to do this.

Reply:
Great suggestion! If we end up wanting to switch between different kinds of SS encoders, `Enum` is a great choice.